Templates/Tests/TemplatesTests.swift

```diff
@@ -82,6 +90,12 @@ class TemplatesTests: QuickSpec {
             expect(generatedFileFilteredLines).to(equal(expectedFileFilteredLines))
         }
 
+#if !canImport(ObjectiveC)
```

Comment:
On Linux, `static func setUp()` is not called, so I needed to manually add this `beforeSuite` to generate files before tests are run.